AlphaWallet/Transfer/Controllers/TransactionConfigurator.swift

```diff
@@ -278,6 +275,10 @@ class TransactionConfigurator {
                 completion(.failure(AnyError(Web3Error(description: "malformed tx"))))
             }
         }
+        /* the node can provide reliable gas limit estimates, this prevents running out of gas or defaulting to an
+        inappropriately high gas limit. This can also be an issue for native transfers which are 21k if send to an EOA
+        address but may be higher if sent to a contract address. */
+        estimateGasLimit()
```

Comment:
If we move it here, and without checking against `gasLimitNotSet`, will this override the user's setting?

Reply:
No, it is set after the function call